### ISSUE TITLE
💡 Proposal - Tooltip Accessibility

### ISSUE THREAD

**Issue by Chaoste** [contentful/forma-36#981]
# Forma 36 contribution proposal

## The problem

In terms of accessibility, native tooltips can be very cumbersome. While Forma36 already offers a beautiful custom Tooltip, it's lacking a few improvements to make it readable for screenreaders.

## The proposed solution

For anyone, interested in a deep dive into a11y and tooltips, I recommend [this article]( https://sarahmhigley.com/writing/tooltips-in-wcag-21/), together with [this codepen](https://cdpn.io/smhigley/debug/KjoerX).

Some basic takeaways for improving our Tooltip are:
- Use `aria-describedby` / `aria-labelledby`
- Use `aria-hidden="true"` if the tooltip is not visible
- Dismiss tooltip with keyboard (ESC key?)

Further, we should extend the `Content recommendations` in the README to include the following things:
- A tooltip only describes the UI control that triggers it.
- There is no essential information in the tooltip.
- Only interactive elements (like a link or button, or at least an element with `tabIndex`) have a tooltip.

## Breaking changes

I believe this doesn't require any breaking changes. This only requires adding HTML attributes, adding a key-up event listener, and updated documentation.

Side note: I stumbled across this issue together with @mshaaban0 and we agreed on filing this issue.
